test/powershell/Dockerfile

```diff
@@ -14,3 +23,23 @@ RUN set -ex; \
 
 SHELL [ "/bin/sh", "-c" ]
 RUN pwsh --version
+
+#--------------------------------------
+# test: powershell
+#--------------------------------------
+FROM build as testb
+
+# renovate: datasource=github-releases lookupName=lukesampson/scoop
+RUN install-tool scoop 2020-11-26
```

Comment:
They do not use git tags anymore, so we maybe need to use git hash?

Reply:
Looks like they're still using tags to me? There's only 1 commit since this tag: https://github.com/lukesampson/scoop/commits/master

Reply:
you are right.